src/Evolution/Executables/GeneralizedHarmonic/EvolveGhBbhCce.hpp

```diff
@@ -0,0 +1,600 @@
+#include "PointwiseFunctions/GeneralRelativity/GeneralizedHarmonic/ConstraintGammas.hpp"
+#include "PointwiseFunctions/GeneralRelativity/IndexManipulation.hpp"
+#include "PointwiseFunctions/GeneralRelativity/Psi4Real.hpp"
+#include "PointwiseFunctions/GeneralRelativity/Ricci.hpp"
+#include "PointwiseFunctions/GeneralRelativity/Tags.hpp"
+#include "PointwiseFunctions/GeneralRelativity/WeylElectric.hpp"
+#include "Time/Actions/AdvanceTime.hpp"
+#include "Time/Actions/ChangeSlabSize.hpp"
+#include "Time/Actions/RecordTimeStepperData.hpp"
+#include "Time/Actions/SelfStartActions.hpp"
+#include "Time/Actions/UpdateU.hpp"
+#include "Time/StepChoosers/Cfl.hpp"
+#include "Time/StepChoosers/Constant.hpp"
+#include "Time/StepChoosers/ErrorControl.hpp"
+#include "Time/StepChoosers/Factory.hpp"
+#include "Time/StepChoosers/Increase.hpp"
+#include "Time/StepChoosers/PreventRapidIncrease.hpp"
+#include "Time/StepChoosers/StepChooser.hpp"
+#include "Time/StepChoosers/StepToTimes.hpp"
+#include "Time/StepControllers/Factory.hpp"
+#include "Time/StepControllers/StepController.hpp"
+#include "Time/Tags.hpp"
+#include "Time/TimeSequence.hpp"
+#include "Time/TimeSteppers/Factory.hpp"
+#include "Time/TimeSteppers/LtsTimeStepper.hpp"
+#include "Time/TimeSteppers/TimeStepper.hpp"
+#include "Time/Triggers/TimeTriggers.hpp"
+#include "Utilities/Blas.hpp"
+#include "Utilities/ErrorHandling/Error.hpp"
+#include "Utilities/ErrorHandling/FloatingPointExceptions.hpp"
+#include "Utilities/Functional.hpp"
+#include "Utilities/ProtocolHelpers.hpp"
+#include "Utilities/TMPL.hpp"
+
+/// \cond
+namespace Frame {
+// IWYU pragma: no_forward_declare MathFunction
+struct Inertial;
+}  // namespace Frame
+namespace PUP {
+class er;
+}  // namespace PUP
+namespace Parallel {
+template <typename Metavariables>
+class CProxy_GlobalCache;
+}  // namespace Parallel
+/// \endcond
+
+// Note: this executable does not use GeneralizedHarmonicBase.hpp, because
+// using it would require a number of changes in GeneralizedHarmonicBase.hpp
+// that would apply only when evolving binary black holes. This would
+// require adding a number of compile-time switches, an outcome we would prefer
+// to avoid.
+struct EvolutionMetavars : CharacteristicExtractDefaults {
+  template <bool DuringSelfStart>
+  struct CceWorldtubeTarget;
+
+  static constexpr size_t volume_dim = 3;
+  static constexpr bool use_damped_harmonic_rollon = false;
+  using initial_data = evolution::NumericInitialData;
+  using system = GeneralizedHarmonic::System<volume_dim>;
+  static constexpr dg::Formulation dg_formulation =
+      dg::Formulation::StrongInertial;
+  using temporal_id = Tags::TimeStepId;
+  static constexpr bool local_time_stepping = true;
+  // Set override_functions_of_time to true to override the
+  // 2nd or 3rd order piecewise polynomial functions of time using
+  // `read_spec_piecewise_polynomial()`
+  static constexpr bool override_functions_of_time = true;
```

Comment:
I think now that the control system is working this should be false for both executables

Reply:
I only changed it for the BBH+CCE executable here. I'll make another PR that changes it for just BBH.